### DISCUSSION TITLE
Should the Go version be updated to latest stable?

### DISCUSSION THREAD

**Discussion by rajkumarGosavi** [go-kit/kit#1270] [Q&A] [answered]
Can we update the go version in go.mod file to latest 1.20 or 1.21?

As there are package level improvement, new packages added for slices, maps. 

Full release note here
https://tip.golang.org/doc/go1.21#language

**Comment by peterbourgon** [comment, marked as answer]
No need. Downstream consumers can specify go1.20 or go1.21 in their go.mod, which will be respected by dependencies like Go kit.
